### PR TITLE
Fix null metdata

### PR DESCRIPTION
**Pull request by eithed**

### CODE REVIEW

File: src/Console/Commands/EventStoreWorker.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ private function makeSerializableEvent(EventRecord $event): JsonEventRecord
         $data->setEventNumber($event->getNumber());
         $data->setData(json_encode($event->getData()));
         $data->setEventStreamId($event->getStreamId());
-        $data->setMetadata(json_encode($this->safeGetMetadata($event)));
+        $data->setMetadata(json_encode($this->safeGetMetadata($event) ?? []));
```
Comment (morrislaptop) [inline src/Console/Commands/EventStoreWorker.php]: Would this be better in the safeGetMatadata function?

**Review by morrislaptop — APPROVED** [review]

File: src/Console/Commands/EventStoreWorker.php (hunk 1 above)
Comment (eithed) [reply src/Console/Commands/EventStoreWorker.php]: Yup - moved

Comment (eithed) [reply src/Console/Commands/EventStoreWorker.php]: Yup - moved

**Comment by morrislaptop** [comment]
@eithed Squash and merge or merge for public repos?

**Pull request merged**